Automattic/jetpack
# Blogroll: Allow non-wpcom sites to be suggested

## Proposed changes:

Currently, Blogroll only allows `wpcom` sites to be suggested by site editors, this PR makes it possible to suggest non-wpcom sites, for instance `www.google.com`.

## Testing instructions:






* Apply and sync this branch to your local environment
* Open your simple site in the editor `wp-admin`
* Add the blogroll block and click on the `plus` button to add another item
* If you type a valid domain, you should be able to select it. Try `www.google.com` or any other valid domain.
* For these non-wpcom sites, we can't allow users to subscribe. Therefore, the subscribe button is invisible for the final users.

<img width="692" alt="image" src="https://github.com/Automattic/jetpack/assets/2653810/a7a9f5a4-2322-4a39-b234-9020e1643cd4">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?
